db/migrate/20190218175833_create_tests.rb

```diff
@@ -0,0 +1,16 @@
+class CreateTests < ActiveRecord::Migration[5.2]
```

Comment:
I think a good alternative for this model's name would be `FitnessTest`. The word `Test` on it's own seems like a dangerous name.